webpack.base.prod.config.js

```diff
@@ -1,15 +1,35 @@
 const SentryWebpackPlugin = require("@sentry/webpack-plugin");
 const packageJson = require("./package.json");
+const webpack = require("webpack");
+
+const plugins = [
+    new webpack.DefinePlugin({
+        "process.env.SENTRY_DSN": JSON.stringify(process.env.SENTRY_DSN || ""),
+        "process.env.SENTRY_ORG": JSON.stringify(process.env.SENTRY_ORG || "nodefactory"),
+        "process.env.SENTRY_PROJECT": JSON.stringify(process.env.SENTRY_PROJECT || "chainguardian"),
+        "process.env.SENTRY_AUTH_TOKEN": JSON.stringify(process.env.SENTRY_AUTH_TOKEN || ""),
+    }),
+];
+
+if(process.env.SENTRY_AUTH_TOKEN) {
+    // Sentry CLI used for uploading releases
+    plugins.push(new SentryWebpackPlugin({
+
+        // sentry-cli configuration
+        authToken: process.env.SENTRY_AUTH_TOKEN,
```

Comment:
I see now. You replace DSN usage with this?

Reply:
this is for sourcemap upload (private value), dsn is for error reporting (public value)